ui/src/version.ts

```diff
@@ -1 +1 @@
-export let version: string = 'v0.6.5';
+export const version: string = 'v0.6.5';
```

Comment:
This is generated from my `deploy.sh` script, but don't worry, I can change that one.